src/sssom/parsers.py

```diff
@@ -828,12 +828,15 @@ def _cell_element_values(cell_node, converter: Converter, mapping_predicates) ->
                 logging.warning(e)
 
     mdict[MAPPING_JUSTIFICATION] = MAPPING_JUSTIFICATION_UNSPECIFIED
-
-    m = Mapping(**mdict)
-    if _is_valid_mapping(m):
-        return m
-    else:
-        return None
+    try:
+        m = Mapping(**mdict)
```

Comment:
Maybe due to some change in LinkML, this line has started throwing an error when the mapping is not wellformed (it also means the below line is not necessary anymore, but we may use it in the future).

In essence, the only thing that happens here is wrapping the `m = Mapping(**mdict)` in try catch so it does not fail hard.

Reply:
If the below line isn't used anymore, let's delete it. Saving stuff for the future is a big programming trap

Reply:
Ok, I added a test now as well that ensures that line _actually_ does as I claim.